dockerfiles/cli/cli.sh

```diff
@@ -35,16 +35,16 @@ cli_init() {
   CODENVY_HOST=${CODENVY_HOST:-${DEFAULT_CODENVY_HOST}}
 
   if [[ "${CODENVY_HOST}" = "" ]]; then
-    info "Welcome to Codenvy!"
+    info "Welcome to $CHE_FORMAL_PRODUCT_NAME!"
     info ""
     info "We did not auto-detect a valid HOST or IP address."
-    info "Pass CODENVY_HOST with your hostname or IP address."
+    info "Pass ${CHE_PRODUCT}_HOST with your hostname or IP address."
```

Comment:
Yes I means until now, we were not using it so maybe we can merge variables. Also in case of che  CHE_PRODUCT_NAME is "ECLIPSE CHE" so it will not work for property name ( uppercase of mini product name will)